packages/data/src/world/mm/great_bay_temple.yml

```diff
@@ -44,7 +44,7 @@
     MAGIC: "true"
   locations:
     "Great Bay Temple SF Central Room Barrel": "true"
-    "Great Bay Temple SF Central Room Underwater Pot": "has_mask_zora || (has_arrows && (has(MASK_GREAT_FAIRY) || underwater_walking_strict))"
+    "Great Bay Temple SF Central Room Underwater Pot": "has_mask_zora || (has(MASK_GREAT_FAIRY) && (has_arrows || (underwater_walking_strict && has_mask_blast)))"
```

Comment:
This isn't right. You dont need great fairy's mask to get the fairy with iron boots, as long as you can break the pot somehow. So overall it should be GFM + Arrows OR Underwater Walking + Blast Mask OR Underwater Walking + Arrows

Reply:
You can't get the fairy even with iron boots. i tried and tried and could not get it despite all the rolling towards the center i did.

If it's possible to get, should most likely be behind a trick.

Reply:
I did try this and it is possible. Definitely annoying though.